### DISCUSSION TITLE
Follow and Execute code that breaks around multiple lines in Python (VSCode)

### DISCUSSION THREAD

**Discussion by henningsway** [quarto-dev/quarto-cli#4372] [Feature Requests]
This is likely not a Quarto core problem, but I haven't found a good solution yet that's why I post here.

I really like, that in RStudio long expressions are executed via a single command, so I don't have to mark the whole expression to run it.

I'd like to achieve the same behaviour with python (using vscode).

**Example**
```{python}
a = [1,  # <- executing this line
     2]
```

results in

>   Cell In[10], line 1
>     a = [1,
>            ^
> SyntaxError: unexpected EOF while parsing
> 

Is there something that Quarto can do here to help? :)

**Comment by mcanouil** [comment]
To note the issue is still there.
Could you open it in <https://github.com/quarto-dev/quarto>?
